src/vs/workbench/contrib/debug/browser/breakpointEditorContribution.ts

```diff
@@ -267,7 +267,9 @@ export class BreakpointEditorContribution implements IBreakpointEditorContributi
 						}
 					} else {
```

Comment:
This should also work for breakpoints with conditions and logpoints, in the block above.

Reply:
Very good point, thank you. I've implemented it in 37ed8a5 and updated the demo recording.

Also I've changed a few variable names in 59b1887. I hope you will find it more readable now.